examples/feature_convert_part/sdf/box.py

```
@@ -0,0 +1,9 @@
+from sdf import *
```

Ruff (0.15.21)

[error] `from sdf import *` used; unable to detect undefined names

(F403)